### PR TITLE
chore(react-chart): fix default tick count calculation

### PR DESCRIPTION
**Pull request by DmitryBogomolov**

### CODE REVIEW

File: packages/dx-chart-core/src/plugins/axis/computeds.test.ts [hunk 1]
```diff
@@ -130,6 +130,24 @@ describe('axisCoordinates', () => {
...
+    it('should generate ticks when pane size is zero, horizontal', () => {
+      (isHorizontal as jest.Mock).mockReturnValue(true);
+      const coordinates = axisCoordinates({
+        scale, tickSize, indentFromAxis, scaleName: 'test-name', position: 'top',
+        paneSize: [0, 0],
+      } as any);
+      expect(scale.ticks).toBeCalledWith(10);
+    });
+
+    it('should generate ticks when pane size is zero, vertical', () => {
```
Comment (Krijovnick) [inline packages/dx-chart-core/src/plugins/axis/computeds.test.ts]: One of the test is excess

Comment (DmitryBogomolov) [reply packages/dx-chart-core/src/plugins/axis/computeds.test.ts]: All tests of **linear** module are paired for horizontal and vertical cases, except for tests for formatting.
Tick count calculation depends on orientation in the sense that `isHor` flag is used to determine which part of the pane size object is used.

Comment (Krijovnick) [reply packages/dx-chart-core/src/plugins/axis/computeds.test.ts]: On each new part of code we write one test. There are no reference about orientation in the changes. And tests with dependence of `paneSize` and orientation has already. Thus we should only check that the function returns the ticks if `paneSize` has zero value. 
By the way, constant `coordinates` is also reduntal.

Comment (DmitryBogomolov) [reply packages/dx-chart-core/src/plugins/axis/computeds.test.ts]: Though there is no orientation reference in the changes, tick calculation depends on the orientation.

```js
const tickCount = getTickCount(scale.range(), paneSize[1 - Number(isHor)]);
```
Should **should pass correct domain to scale, vertical** test be removed?

Comment (Krijovnick) [reply packages/dx-chart-core/src/plugins/axis/computeds.test.ts]: I think it is not matter vertical or horizontal test will be removed. In both cases we will be in the same part of code.

Comment (DmitryBogomolov) [reply packages/dx-chart-core/src/plugins/axis/computeds.test.ts]: I meant that there are two **should pass correct domain to scale** tests. And it one of tests in this **PR** is to be removed then one of **pass correct domain** tests should be removed as well.
Shouldn't it?

Comment (Krijovnick) [reply packages/dx-chart-core/src/plugins/axis/computeds.test.ts]: yes

Comment (DmitryBogomolov) [reply packages/dx-chart-core/src/plugins/axis/computeds.test.ts]: Let it be so then.

**Pull request merged**